Tests/SonarScanner.MSBuild.PreProcessor.Test/ArgumentProcessorTests.cs

```diff
@@ -217,7 +217,25 @@ public void PreArgProc_Region_Invalid()
         @"The arguments 'sonar.host.url' and 'sonar.scanner.sonarcloudUrl' are both set. Please set only 'sonar.scanner.sonarcloudUrl'.")]
     [DataRow(null, "https://host", null, "https://api", typeof(ServerHostInfo), "https://host", "https://api")]
     [DataRow(null, "https://host", null, null, typeof(ServerHostInfo), "https://host", "https://host/api/v2")]
-    public void PreArgProc_Region_Overrides(string region, string hostOverride, string sonarClourUrlOverride, string apiOverride, Type expectedHostInforType, string expectedHostUri, string expectedApiUri, params string[] expectedWarnings)
+
+    [DataRow(null, "https://sonarqube.us", null, null, typeof(CloudHostInfo), "https://sonarqube.us", "https://api.sonarqube.us")]
+    [DataRow(null, "https://SONARQUBE.us/", null, null, typeof(CloudHostInfo), "https://SONARQUBE.us/", "https://api.sonarqube.us")]
```

Comment:
As the trailing `/` and different cases are already tested in `ProcArgs_HostUrl_SonarCloudUrl_HostUrlIsAnySonarCloud`, I would not duplicate those tests here. 

Here, I would focus on the different combinations of different values set/notset to make it more clear what we test.

Reply:
I kept the capitalized versions, so you can see what url comes from the parameter and what was taken from the defaults.